src/rules/sort-keys.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @fileoverview Rule to require JSON object keys to be sorted. Cribbed largely from https://github.com/eslint/eslint/blob/main/lib/rules/sort-keys.js
+ * @author Robin Thomas
+ */
+
+import naturalCompare from "natural-compare";
+
+const comparators = {
+	ascending: {
+		alphanumeric: {
+			sensitive: (a, b) => a <= b,
+			insensitive: (a, b) => a.toLowerCase() <= b.toLowerCase(),
+		},
+		natural: {
+			sensitive: (a, b) => naturalCompare(a, b) <= 0,
+			insensitive: (a, b) =>
+				naturalCompare(a.toLowerCase(), b.toLowerCase()) <= 0,
+		},
+	},
+	descending: {
+		alphanumeric: {
+			sensitive: (a, b) =>
+				comparators.ascending.alphanumeric.sensitive(b, a),
+			insensitive: (a, b) =>
+				comparators.ascending.alphanumeric.insensitive(b, a),
+		},
+		natural: {
+			sensitive: (a, b) => comparators.ascending.natural.sensitive(b, a),
+			insensitive: (a, b) =>
+				comparators.ascending.natural.insensitive(b, a),
+		},
+	},
+};
+
+export default {
+	meta: {
+		defaultOptions: [
+			"asc",
+			{
+				allowLineSeparatedGroups: false,
+				caseSensitive: true,
+				minKeys: 2,
+				natural: false,
+			},
+		],
+
+		docs: {
+			description: `Require JSON object keys to be sorted`,
+		},
+
+		messages: {
+			sortKeys:
+				"Expected object keys to be in {{sortName}} case-{{sensitivity}} {{direction}} order. '{{thisName}}' should be before '{{prevName}}'.",
+		},
+
+		schema: [
+			{
+				enum: ["asc", "desc"],
+			},
+			{
+				type: "object",
+				properties: {
+					caseSensitive: {
+						type: "boolean",
+					},
+					natural: {
+						type: "boolean",
+					},
+					minKeys: {
+						type: "integer",
+						minimum: 2,
+					},
+					allowLineSeparatedGroups: {
+						type: "boolean",
+					},
+				},
+				additionalProperties: false,
+			},
+		],
+
+		type: "suggestion",
+	},
+
+	create(context) {
+		const [directionShort, { caseSensitive, natural, minKeys }] =
```

Comment:
`allowLineSeparatedGroups` not implemented?

Reply:
Oops, I added the logic for it, but forgot to add in the option. It's odd that all the tests pass without it. Will look into that today. Thanks!

Reply:
Updated